index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Static Web HTML &amp; CSS Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<article class="product-card">
+		<header class="product-title">
+			<h1>Product Title</h1>
+		</header>
+		<section class="product-info">
+			<img src="img/logo.png" alt="Product image">
+			<div class="product-description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+			<div class="availability">Available</div>
+		</section>
+		<section class="product-specs">
+			<div class="product-specs-content">
+				<header class="spec-header">
+					Specifications
+				</header>
```

Comment:
Consider inlining short tags like this: `<header class="spec-header">Specifications</header>`